contracts/eosio.token/include/eosio.token/eosio.token.hpp

```diff
@@ -126,7 +126,7 @@ namespace eosio {
          static asset get_supply( const name& token_contract_account, const symbol_code& sym_code )
          {
             stats statstable( token_contract_account, sym_code.raw() );
-            const auto& st = statstable.get( sym_code.raw(), "no supply object found" );
+            const auto& st = statstable.get( sym_code.raw(), "Wrong symbol type" );
```

Comment:
Having the same error message as `get_balance` would make it difficult to troubleshoot when distinguishing between which method the error came from.

- Specifying the word "supply" would be beneficial.
- The word "Wrong" isn't commonly used in error messages.
- All other error messages in `eosio.token` are lowercase
- Could specify it's "symbol code" instead of "symbol type" (they are different types)

Perhaps a better error message handling could be:

"invalid supply symbol code"